new-style.css

```diff
@@ -726,6 +726,51 @@ html[dir="rtl"] .footer .navs {
     text-decoration: underline;
 }
 
+/*Covid*/
+.covid {
+    background-color:#5BC6F9;
+    position: relative;
+  }
+  
+.covid h1 {
+    color:#fff;
+    text-align: start;
+    padding-top: 2em;
+    padding-inline-start: 6.25em;
+    padding-inline-end: 14.5em;
+}
+
+.covid .covid-content .covid-container {
+    display: flex;
+    height: 9.5em;
+  }
+   
+.covid .covid-content .covid-container p {
+    color:#fff;
+    padding-inline-start: 12.5em;
+    padding-inline-end: 14.5em;
+    padding-top: 1em;
+  }
+  
+.covid .imageCovid {
+    width: 9.25em;
+    height: 9.25em;
+    position: absolute;
+    right: 0;
+    bottom: 0;
+    align-items: flex-end;
+  }
+  
+html[dir='rtl'] .covid .imageCovid {
+    width: 9.25em;
```

Comment:
You're not changing `width`, `height`, `align-items` and `bottom` so why repeat them?